src/istgt_lu.h

```diff
@@ -138,27 +139,31 @@ typedef enum {
  */
 #define ISTGT_MAX_LU_RUNNING_STATE_RETRY_COUNT 45
 
-typedef enum {
+typedef enum
```

Comment:
these changes are not required as cstyle is not showing any error here, and, in the code, 'flower braces' should be given in the same line of 'enum', 'struct', 'if', 'for' etc.